tests/Unit/OAuth/Infrastructure/Provider/StatefulFacebookProviderDouble.php

```
@@ -0,0 +1,44 @@
+    {
+        parent::setPkceCode($pkceCode);
+
+        return $this;
+    }
+
+    /**
+     * @param string $grant
+     * @param array<string, string> $params
+     */
+    #[\Override]
+    public function getAccessToken($grant = 'authorization_code', array $params = []): AccessToken
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$grant'. (undefined)

(UnusedFormalParameter)